package.json

```diff
@@ -20,8 +20,8 @@
         "eslint-plugin-react": "3.16.1"
     },
     "dependencies": {
-        "@dicetechnology/react-native-dice-video": "git+ssh://git@github.com:DiceTechnology/react-native-dice-video.git#6.43.6",
-        "@dicetechnology/dice-unity": "^2.26.3",
+        "@dicetechnology/react-native-dice-video": "git+ssh://git@github.com:DiceTechnology/react-native-dice-video.git#6.48.0",
+        "@dicetechnology/dice-unity": "https://github.com/DiceTechnology/dice-unity.git#8a16345",
```

Comment:
Let's remember to revert this

Reply:
I think we need to bump the dice-unity or revert it, instead of use commit hash in a PR.